### PR TITLE
feat(api): basic implementation of create submission endpoint

### PR DESCRIPTION
**Pull request by JamesP-KC**
- basic implementation of `POST /api/v1/submission/{caseRef}` with a mocked response that echos back the data sent in the request

### CODE REVIEW

File: packages/applications-service-api/src/routes/index.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const dbConnectionTestRouter = require('./db-connection-test');
 const documentRouter = require('./documents');
 const representationsRouter = require('./representations');
 const timetablesRouter = require('./timetables');
+const submissionRouter = require('./submission');
```
Comment (Philip-Philippou1) [inline packages/applications-service-api/src/routes/index.js:14]: Not important but should this be submissions in line with the endpoint name.

Comment (JamesP-KC) [reply packages/applications-service-api/src/routes/index.js:14]: Yes good spot, when creating these new files I had `submission` in my head but you are right it is `/submissions`. Will update

File: packages/applications-service-api/src/controllers/submission.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		res.status(200).send({
+			name: req.body.name,
+			email: req.body.email,
+			interestedParty: parseBoolean(req.body.interestedParty),
+			ipReference: req.body.ipReference,
+			deadline: req.body.deadline,
+			submissionType: req.body.submissionType,
+			sensitiveData: parseBoolean(req.body.sensitiveData),
+			lateSubmission: parseBoolean(req.body.lateSubmission),
+			submissionId: req.body.submissionId,
```
[the remark this answers is not in the document]
Comment (JamesP-KC) [reply packages/applications-service-api/src/controllers/submission.js:17]: sounds good, will add your suggestion in 👍

**Review by Philip-Philippou1 — APPROVED** [review]

**Pull request merged**